adopt-a-dog/dogs.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styling/stylesheet.css">
+  </head>
+  <body>
+    <header>
+        <div class="page-banner">
+           <div class="page-banner-title">
+             <img src="images/adoptadog-logo.svg" alt="Adopt a Dog Logo" class="logo-top">
+             <h1>Seattle's premiere dog adoption agency</h1>
+           </div>
+           <div class="page-banner-Link">
+            <img src="images/cart-icon.svg" alt="Shopping icon" class="shopping-icon">
+            <div class="banner-price">
+              <p>$1234.56</p>
+            </div>
+            <form  action="check-out.html">
+              <input type="submit" class="checkout-button-top" value="Checkout">
+            </form>
+           </div>
+        </div>
+        <nav class="nav-bar-container">
+          <ul class="nav-bar">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="dogs.html">Dogs</a></li>
+            <li><a href="blog.html">Blog</a></li>
+          </ul>
+        </nav>
+    </header>
+    <main>
+      <div class="wrapper">
+        <h1 class = "page-title" >Here are all of our Dogs!</h1>
+        <div class="picture-flex">
+          <section class="dog-info">
+            <img src="images/alex-lacayo-756508-unsplash.jpg" alt="Murphy Dog Picture" class="dog-img">
+            <h3>Murphy</h3>
+            <h5><b>Cost to Adopt:</b> $123.45</h5>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+            <a href="check-out.html"><button type="button" class="checkout-button">Checkout</button></a>
+          </section>
+          <section class="dog-info">
+            <img src="images/nikolay-tchaouchev-785309-unsplash.jpg" alt="Poppy Dog Picture" class="dog-img">
+            <h3>Poppy</h3>
+            <h5><b>Cost to Adopt:</b> $123.45</h5>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+            <a href="check-out.html"><button type="button" class="checkout-button">Checkout</button></a>
```

Comment:
In regards to your question....

This shouldn't link to the checkout page. What this will do is just "adopt" the dog and add it to our cart. You can take out the anchor tags, and just use a button.